xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -71,6 +71,9 @@ bool CPVRChannelGroups::Update(const CPVRChannelGroup &group, bool bUpdateFromCl
     if (group.GroupID() > 0)
       updateGroup = GetById(group.GroupID());
 
+    if (!updateGroup && group.IsInternalGroup() && (updateGroup = GetGroupAll()) && bUpdateFromClient)
```

Comment:
As I understood right, you never want that the Dummy name can overwrite the value from db.
If I would use the simple form above the db name will overwritten / updated with the dummy name if the Update() is called from client.